Cargo.toml

```diff
@@ -18,27 +18,44 @@ path = "src/lib.rs"
 
 [dependencies]
 serde = { version = "1.0", features = ["derive"] }
+serde_json = { version = "1.0" }
 bitcoin = { version = "0.31.0", features = ["serde", "std"], default-features = false }
 hex = { package = "hex-conservative", version = "*" }
 log = "^0.4"
 minreq = { version = "2.11.0", features = ["json-using-serde"], optional = true }
 reqwest = { version = "0.11", optional = true, default-features = false, features = ["json"] }
+hyper = { version = "0.14", optional = true, features = ["http1", "client", "runtime"], default-features = false }
+arti-hyper = { version = "0.8.3", optional = true, features = ["default"] }
+arti-client = { version = "0.8.3", optional = true }
+tor-rtcompat = { version = "0.8.2", optional = true, features = ["tokio"]}
+tls-api = { version = "0.9.0", optional = true }
+tls-api-native-tls = { version = "0.9.0", optional = true }
+ahash = { version = "=0.8.6", optional = true } # ahash 0.8.7 version don't work with our MSRV on aarch64, check: https://github.com/tkaitchuck/aHash/issues/195
+
+[target.'cfg(target_vendor="apple")'.dependencies]
+tls-api-openssl = { version = "0.9.0", optional = true }
 
 [dev-dependencies]
-serde_json = "1.0"
 tokio = { version = "1.20.1", features = ["full"] }
 electrsd = { version = "0.27.0", features = ["legacy", "esplora_a33e97e1", "bitcoind_25_0"] }
 lazy_static = "1.4.0"
 
 [features]
-default = ["blocking", "async", "async-https"]
+default = ["blocking", "async", "async-https", "async-arti-hyper"]
+
 blocking = ["minreq", "minreq/proxy"]
 blocking-https = ["blocking", "minreq/https"]
 blocking-https-rustls = ["blocking", "minreq/https-rustls"]
 blocking-https-native = ["blocking", "minreq/https-native"]
 blocking-https-bundled = ["blocking", "minreq/https-bundled"]
+
 async = ["reqwest", "reqwest/socks"]
 async-https = ["async", "reqwest/default-tls"]
 async-https-native = ["async", "reqwest/native-tls"]
 async-https-rustls = ["async", "reqwest/rustls-tls"]
 async-https-rustls-manual-roots = ["async", "reqwest/rustls-tls-manual-roots"]
+
+# TODO: (@leonardo) Should I rename it to async-anonymized ?
```

Comment:
how about `async-tor` instead of `async-anonymized` ? It's shorter and most users know what `tor` is but may not know about `arti`.